build/root/BUILD.root

```diff
@@ -5,7 +5,7 @@
 
 package(default_visibility = ["//visibility:public"])
 
-load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_prefix")
+load("@io_bazel_rules_go//go:def.bzl", "go_prefix")
```

Comment:
Hi @ixdy, 

Does this PR automatically fix `./hack/update-bazel.sh`, this seem to include `go_library` back.

Reply:
it shouldn't be doing that. Do you have a `generated.pb.go` and/or `types.go` file in the root of your kubernetes tree? I wonder if one of the code generation steps is interacting poorly.